### DISCUSSION TITLE
GSoC idealist was editable by anyone

### DISCUSSION THREAD

**Discussion by hellosagar** [robolectric/robolectric#7107] [General]
I think it is supposed to be read-only?

**Comment by utzcoz** [comment]
@hellosagar Thanks for reporting this problem. Looks like it is read-only for me when I view it anonymous although its link with edit suffix. And I have checked its settings, and link is read-only for everyone except some editors. But I actually found intermediate wrong format of one page today, and it restored to correct format very soon. Hi @hoisie , could you help to double-check slide permission? Thanks.

**Comment by hellosagar** [comment]
@utzcoz nice, yeah now it's read-only. I just double checked

**Reply by utzcoz** [reply]
Thanks for double-checking it. :)

**Comment by hoisie** [comment]
Thanks to whoever fixed this issue :)
